ProgramTests/CalculatorTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace Calculate.Tests
+{
+    [TestClass]
+    public class CalculatorTests
+    {
+        Calculator calc = new();
+        [TestMethod]
+        public void TestDictionaryContents()
+        {
+            Func<int, int, double> addition = Calculator.Add;
+            Func<int, int, double> subtraction = Calculator.Subtract;
+            Func<int, int, double> multiplication = Calculator.Multiply;
+            Func<int, int, double> division = Calculator.Divide;
+            Assert.AreEqual<Func<int, int, double>>(addition, calc.MathematicalOperators['+']);
+            Assert.AreEqual<Func<int, int, double>>(subtraction, calc.MathematicalOperators['-']);
+            Assert.AreEqual<Func<int, int, double>>(multiplication, calc.MathematicalOperators['*']);
+            Assert.AreEqual<Func<int, int, double>>(division, calc.MathematicalOperators['/']);
+        }
+
+        [TestMethod]
+        public void TestAdd()
+        {
+            double testAddAct = Calculator.Add(1, 2);
+            double testAddExp = 3;
+            Assert.AreEqual<double>(testAddExp, testAddAct);
+        }
+
+        [TestMethod]
+        public void TestSubtract()
+        {
+            double testSubtractAct = Calculator.Subtract(1, 2);
+            double testSubtractExp = -1;
+            Assert.AreEqual<double>(testSubtractExp, testSubtractAct);
+        }
+
+        [TestMethod]
+        public void TestMultiply()
+        {
+            double testMultiplyAct = Calculator.Multiply(3, 5);
+            double testMultiplyExp = 15;
+            Assert.AreEqual<double>(testMultiplyExp, testMultiplyAct);
+        }
+
+        [TestMethod]
+        public void TestDivide()
+        {
+            double testDivideAct = Calculator.Divide(5, 2);
+            double testDivideExp = 2.5;
+            Assert.AreEqual<double>(testDivideExp, testDivideAct);
+        }
+
+        [TestMethod]
+        public void TestTryCalculateAddSuccess()
```

Comment:
```suggestion
        public void TryCalculate_ValidAddExpression_Success()
```
Placing "Test" into the name of each test method isn't ideal.
There are different commonly used formats, but the following is the most clear to me:
[UnitOfWork_StateUnderTest_ExpectedBehavior]

Where UnitOfWork refers to the member you're testing. Which, in this case, is the TryCalculate() method.